symfony/panther/1.0/manifest.json

```
@@ -19,6 +19,20 @@
             "content": "        <extension class=\"Symfony\\Component\\Panther\\ServerExtension\" />",
             "position": "after_target",
             "target": "<extensions>",
+            "warn_if_missing": false
+        },
+        {
+            "file": "phpunit.dist.xml",
+            "content": "        <bootstrap class=\"Symfony\\Component\\Panther\\ServerExtension\" />",
+            "position": "after_target",
+            "target": "<extensions>",
+            "warn_if_missing": false
+        },
+        {
+            "file": "env.test",
```

Hey @alexander-schranz 👋 

Is this missing a dot in the file name maybe?

```suggestion
            "file": ".env.test",
```